addons/product/views/product_template_views.xml

```diff
@@ -114,8 +114,8 @@
                             </span>
                             <field name="product_properties" widget="properties"/>
                         </main>
-                        <aside class="col-2 p-0">
-                            <field name="image_128" widget="image" alt="Product"/>
+                        <aside class="flex-shrink-0">
```

Comment:
Why `flex-shrink-0` is needed? I think that's just because  of `w-100`  applied on `<main>`, but it looks unnecessary as well :)

Reply:
> If flex-shrink-0 is not defined, the width could be reduced according to the main content even without w-100.

.. meaning "_`aside` is broken unless we use this class in each template_" .. If it's the case it's better to add the rule in CSS instead 😉

In a previous comment I've proposed to remove `grow = 1` and  test flex-basis  ( https://github.com/odoo/odoo/pull/169790#discussion_r1645972279) ... I think you applied my suggestion without testing it and probably applied `shrink-0`after to overcome the undesired results.
  
  If `shrink` was still necessary you could have keep it rather than replacing it with utility classes 😉

Reply:
Ok I see your point 👍.
I've removed these utility classes in favor of global css.